.github/pull_request_template.md

```diff
@@ -16,18 +16,26 @@ Any User Interface Changes (namelist or namelist defaults changes)?
  - [ ] No
 
 Testing performed if application target is CESM:(either UFS-S2S or CESM testing is required):
```

Comment:
Do we want to change UFS-s2s to UFS coupled testing in the future since the UFS tests covers both weather and s2s?

Reply:
Good point. I'll make a note; she said she has an upcoming PR also so she could do it then.